src/AbstractWebApplication.php

```diff
@@ -634,6 +634,25 @@ protected function getHttpStatusValue($value)
 	}
 
 	/**
+	 * Check if the value is a valid HTTP 1.1 status code
+	 *
+	 * @param   int  $code  The given status as int or string
+	 *
+	 * @return  bool
+	 *
+	 * @since  __DEPLOY_VERSION__
+	 */
+	protected function isValidHttpStatus($code)
+	{
+		if (array_key_exists($code, $this->responseMap))
```

Comment:
`return array_key_exists($code, $this->responseMap);`

No need to manually specify true/false returns.